run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon

```diff
@@ -115,9 +113,7 @@ Mask fractions:              false
     --> CMIP6_SFC_CH4          :       false    # 1750-1978
 # ..... Ship emissions ...............
     --> SHIP                   :       true
-    --> CEDSv2_SHIP            :       true     # 1750-2017
-    --> CEDS_GBDMAPS_SHIP      :       false    # 1970-2017
-    --> CEDS_GBDMAPS_SHIP_byFuelType:  false    # 1970-2017
+    --> CEDS_01x01_SHIP        :       ${RUNDIR_USE_CEDS}    # 1980-2019
```

Comment:
This should be "true" based on previous version

Reply:
Now resolved in 1ef383b65d4a54249b1421d5fa3049cfc7ddb912